docs/visual-basic/programming-guide/concepts/linq/how-to-populate-object-collections-from-multiple-sources-linq.md

```diff
@@ -59,7 +59,7 @@ Class PopulateCollection
                           Let splitScoreLine = scoreLine.Split(New Char() {","})
                           Where Convert.ToInt32(splitName(2)) = Convert.ToInt32(splitScoreLine(0))
                           Select New Student() With {
-                               .FirstName = splitName(0), .LastName = splitName(1), .ID = splitName(2),
+                               .FirstName = splitName(1), .LastName = splitName(0), .ID = splitName(2),
                                .ExamScores = (From scoreAsText In splitScoreLine Skip 1
                                              Select Convert.ToInt32(scoreAsText)).ToList()}
 
```

Comment:
The output from the example also needs to change, since it displays last name before first name.

Reply:
Got it. But I did this from the docs (clicking through the buttons from docs), so I have no clue where my copy of the code to change is. How does that work?

Reply:
@KathleenDollard, you can continue to use the GitHub editor. Select the **Files changed** tab, then click the pencil icon to edit the file. When you're finished, make sure that the **Commit directly to the patch-3 branch** option is selected, then select **Commit changes**.

Reply:
Thanks, the output has been updated.